django_openid_auth/views.py

```diff
@@ -40,7 +40,10 @@
 from django.contrib.auth import (
     REDIRECT_FIELD_NAME, authenticate, login as auth_login)
 from django.contrib.auth.models import Group
-from django.core.urlresolvers import reverse
+try:
```

Comment:
Can you mark this as a shim with a comment, so we'll know when to remove it? Something like:

`# SHIM: For Django 1.11 and 2.x compatiblity, can remove when support for 1.11 is dropped.`

Reply:
@bmedx Thanks for the suggestion. shim added now!